project-euler/task1/task1.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Multiples Sum</title>
+</head>
+<body>
+
+  <h1>Sum of Multiples of 3 or 5 below 1000</h1>
+
+  <script>
+    function findMultiplesSum(limit) {
+      let sum = 0;
+
+      for (let i = 1; i < limit; i++) {
+        if (i % 3 === 0 || i % 5 === 0) {
+          sum += i;
+        }
+      }
+
+      return sum;
+    }

```

Comment:
We can move this to a separate .js file

Reply:
Created a script.js file seperately and moved the code .